### PR TITLE
AP_GPS: change GPS_TYPE value 10 from SBF to Septentrio

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by tpwrules** [comment]
GPS_TYPE 26 should be updated too?

**Comment by Ryanf55** [comment]
Can do Trimble-GSOF too.

**Review by Ryanf55 — APPROVED** [review]

File: libraries/AP_GPS/AP_GPS_Params.cpp [hunk 1]
```diff
@@ -25,7 +25,7 @@ const AP_Param::GroupInfo AP_GPS::Params::var_info[] = {
     // @Param: TYPE
     // @DisplayName: GPS type
     // @Description: GPS type
-    // @Values: 0:None,1:AUTO,2:uBlox,5:NMEA,6:SiRF,7:HIL,8:SwiftNav,9:DroneCAN,10:SBF,11:GSOF,13:ERB,14:MAVLink,15:NOVA,16:HemisphereNMEA,17:uBlox-MovingBaseline-Base,18:uBlox-MovingBaseline-Rover,19:MSP,20:AllyStar,21:ExternalAHRS,22:DroneCAN-MovingBaseline-Base,23:DroneCAN-MovingBaseline-Rover,24:UnicoreNMEA,25:UnicoreMovingBaselineNMEA,26:SBF-DualAntenna
+    // @Values: 0:None,1:AUTO,2:uBlox,5:NMEA,6:SiRF,7:HIL,8:SwiftNav,9:DroneCAN,10:Septentrio,11:GSOF,13:ERB,14:MAVLink,15:NOVA,16:HemisphereNMEA,17:uBlox-MovingBaseline-Base,18:uBlox-MovingBaseline-Rover,19:MSP,20:AllyStar,21:ExternalAHRS,22:DroneCAN-MovingBaseline-Base,23:DroneCAN-MovingBaseline-Rover,24:UnicoreNMEA,25:UnicoreMovingBaselineNMEA,26:SBF-DualAntenna
```
Comment (tridge) [inline libraries/AP_GPS/AP_GPS_Params.cpp:28]: maybe Septentrio(SBF) and Trimble(GSF) Septentrio-DualAntenna(SBF)

Comment (Ryanf55) [reply libraries/AP_GPS/AP_GPS_Params.cpp:28]: Trimble(GSOF)

Comment (peterbarker) [reply libraries/AP_GPS/AP_GPS_Params.cpp]: Done

**Pull request merged**